webdriver/tests/bidi/storage/set_cookie/partition.py

```diff
@@ -40,19 +33,14 @@ async def test_partition_context_frame(bidi_session, set_cookie, top_context, te
     all_contexts = await bidi_session.browsing_context.get_tree(root=top_context["context"])
     frame_context_id = all_contexts[0]["children"][0]["context"]
 
-    partition = BrowsingContextPartitionDescriptor(frame_context_id)
-
     set_cookie_result = await set_cookie(
-        cookie=create_cookie(domain=domain_value()),
-        partition=partition)
+        cookie=create_cookie(domain=root_page_domain),
+        partition=(BrowsingContextPartitionDescriptor(frame_context_id)))
 
-    assert set_cookie_result == {
-        'partitionKey': {
-            'sourceOrigin': frame_source_origin
-        },
-    }
+    # Browsing context does not require a `sourceOrigin` partition key.
+    assert set_cookie_result == {'partitionKey': {}, }
```

Comment:
does it still allow a sourceOrigin? I think there might be a sourceOrigin here or not based on what browser is under test as there is currently no agreement on partitioning rules.

Reply:
Good point. Added assertion allowing result to be extensible.